### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by kberzinch**

### CODE REVIEW

**Review by JoshOldenburg — APPROVED** [review]

File: loop_workday_upload.py [hunk 1]
```diff
@@ -0,0 +1,499 @@
...
+
+        if workday_attachment_response.status_code != 200:
+            print(workday_attachment_response.status_code)
+            print(workday_attachment_response.text)
+            raise ValueError("Unexpected response code from Workday")
+
+        print("Uploading attachment " + attachment + " to Loop")
+
+        loop_attachment_response = post(
+            url=server + "/api/v1/workday/attachments/" + attachment,
```
Comment (evan10s) [inline loop_workday_upload.py:278]: nit: I prefer f-strings for concatenation like this

File: loop_workday_upload.py [hunk 2]
```diff
@@ -0,0 +1,499 @@
...
+            headers={
+                "Authorization": "Bearer " + token,
```
Comment (evan10s) [inline loop_workday_upload.py:280]: Same here

**Review by evan10s — APPROVED** [review]
Lgtm, besides my preference for f-strings which is not a big deal

File: loop_workday_upload.py (hunk 1 above)
Comment (kberzinch) [reply loop_workday_upload.py]: Good call, updated everywhere

**Pull request merged**